inventories/user_plugins/inventory_plugins/fox.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+    inventory: cow
+    version_added: "2.7"
+    short_description: What does the fox say? No one knows, error!
+    description:
+        - Ignores whatever you give it
+        - You will never find out what the fox says
+'''
+
+EXAMPLES = r'''
+    # plugin: fox
+'''
+
+from ansible.plugins.inventory import BaseInventoryPlugin
+
+
+def ancient_mystery():
+    raise Exception('Gering-ding-ding-ding-dingeringeding')
```

Comment:
@kdelee this serves a very specific testing purpose.

```
 [WARNING]:  * Failed to parse /Users/alancoding/Documents/repos/jlaska-ansible-playbooks/inventories/user_plugins/fox.yaml with auto plugin: Gering-ding-ding-ding-dingeringeding

  File "/Users/alancoding/Documents/repos/ansible/lib/ansible/inventory/manager.py", line 272, in parse_source
    plugin.parse(self._inventory, self._loader, source, cache=cache)
  File "/Users/alancoding/Documents/repos/ansible/lib/ansible/plugins/inventory/auto.py", line 55, in parse
    plugin.parse(inventory, loader, path, cache=cache)
  File "/Users/alancoding/Documents/repos/jlaska-ansible-playbooks/inventories/user_plugins/inventory_plugins/fox.py", line 43, in parse
    ancient_mystery()
  File "/Users/alancoding/Documents/repos/jlaska-ansible-playbooks/inventories/user_plugins/inventory_plugins/fox.py", line 29, in ancient_mystery
    raise Exception('Gering-ding-ding-ding-dingeringeding')
```

So, we need to run this and verify that `ancient_mystery` is in the output.